templates/components/alert.html

```diff
@@ -6,22 +6,22 @@
       <div class="alerts-icon__container alerts-icon__container--48">
         {{ alerts_icon(height=48, alert_active=alert.is_current) }}
         <h2 class="alerts-alert__title govuk-heading-s govuk-!-margin-bottom-3">
-          {% if alert.channel == 'operator' %}
-            Mobile network operator test
-          {% else %}
+          {% if alert.is_public %}
             <span class="govuk-visually-hidden">Emergency alert sent to </span>{{ alert.area_names | formatted_list(before_each='', after_each='') }}
+          {% else %}
+            Mobile network operator test
```

Comment:
I'm cautious that for an alert sent on the `test` channel, that we describe it as a Mobile network operator test.

Example messages that might be sent on the test channel could be a monthly test message to a small area of the UK (it might change which area got it). 

The two risks are
- the mobile network operators aren't happy with us describing them as the ones testing when it is actually us
- people who receive a message on the test channel are confused. For example, the page on mobile network operator tests says they are only for android phone users when technically an iphone user can opt in to messages on the test channel. 

I think this is tolerable for the moment though. When we send our next test channel message (whenever that might be), hopefully we will be looking at gov.uk/alerts and will spot if things look strange and can change it for the next one.

Reply:
Thanks for elaborating on the risks. I did agree this with @quis and as you say, the risk is low for now. My main concern was doing something to display the alerts properly if we get them: without this change they'll get shown like severe / government alerts, but without their own page (since they're not "public").